packages/components/src/ButtonGroup/_docs/ButtonGroup.stories.tsx

```diff
@@ -15,6 +16,18 @@ export default {
   component: ButtonGroup,
   argTypes: { ...classNameOverrideArgType },
   parameters: {
+    a11y: {
+      config: {
+        rules: [
+          ...globalA11yRules,
```

Comment:
Had to do this annoying thing because as soon as you set this config on the story level, it overrides the global one 😕